docs/tutorial.md

```diff
@@ -102,3 +124,9 @@ model environment that you’ve created using the following command.
 ```
 juju destroy-model wordpress-tutorial --destroy-storage
 ```
+
+If you used Multipass, to remove the Multipass instance you created for this tutorial, use the following command.
```

Comment:
Nitpick: We may want to tell the reader to first `exit` from the VM before removing the Multipass instance.